Library/src/Web/EmscriptenWebClient/EmscriptenWebClient.cpp

```diff
@@ -101,6 +101,20 @@ void OnFetchSuccessOrError(emscripten_fetch_t* Fetch)
 	emscripten_fetch_close(Fetch);
 }
 
+void OnFetchError(emscripten_fetch_t* Fetch)
+{
+	auto* Request = reinterpret_cast<csp::web::HttpRequest*>(Fetch->userData);
+
+	if (Request->Retry())
+	{
+		CSP_LOG_WARN_MSG("Retrying failed emscripten request\n");
```

Comment:
Can we put this on the verbose level? Emitting to regular log level feels like it risks being spammy

Reply:
No that's perfect, sorry I misread!